cfme/tests/cloud/test_keypairs.py

```diff
@@ -73,41 +78,54 @@ def test_keypair_crud_with_key(provider, appliance):
 @pytest.mark.tier(3)
 def test_keypair_create_cancel(provider, appliance):
     """ This will test cancelling on adding a keypair
-
-    Steps:
-        * Provide Keypair name.
-        * Select Cloud Provider.
-        * Also delete it.
-
     Polarion:
-        assignee: mnadeem
+        assignee: mmojzis
         casecomponent: WebUI
+        caseimportance: low
         initialEstimate: 1/4h
+        testSteps:
+            1. Cancel creating keypair.
     """
     keypair = appliance.collections.cloud_keypairs.create(
         name="",
         provider=provider,
         cancel=True
     )
-
     assert not keypair.exists
 
 
+@pytest.mark.provider([EC2Provider], override=True, scope="module", selector=ONE_PER_TYPE)
+@pytest.mark.tier(3)
+def test_keypair_create_validation(provider, appliance):
+    """ This will test validating the new of name of the key pair
```

Comment:
@mmojzis Could this be a bit more specific here? This test is really testing the validation that a key pair cannot be added with an empty name.

Reply:
Done.